progressbar_printer_test.go

```diff
@@ -236,18 +237,40 @@ func TestProgressbarPrinter_OutputToWriters(t *testing.T) {
 
 	for testTitle, testCase := range testCases {
 		t.Run(testTitle, func(t *testing.T) {
-			stderr, err := testza.CaptureStderr(func(w io.Writer) error {
```

Comment:
this was flaking with some odd file descriptor error. idk why but it seemed related to testza and not the test itself. Changed to use the internal method

Reply:
That's actually quite funny. The problem here is that Linux and macOS run out of available file descriptors, because PTerm used it so many times. Testza is actually capturing the output by replacing stdout temporarily. This was needed, when we didn't have the option to change the output `Writer` of the printers. That's also actually the reason, why we run all our CI tests on Windows. We might change that to a matrix test again, if it works now :)

Reply:
Interesting, I am macOS but I also have the file descriptors really bumped up in my terminal because it has been an issue in other projects.